modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -261,7 +261,7 @@ export async function getObjectFieldsByExternalReferenceCode(
 	);
 }
 
-export async function getObjectFieldsById(objectDefinitionId: number) {
+export async function getObjectDefinitionObjectFields(objectDefinitionId: number) {
```

Comment:
@igor-franca @MarinhoFeliphe we should keep the "ById" at the end of the name because we have another function to retrieve Object Fields by ERC. Also, I think we shoud apply the same pattern of adding "ObjectDefinition" in the name of that function  as well.

Reply:
In this case, @MarinhoFeliphe  explained to me that it is for entities that are inserted within other entities. We must use the name of the parent entity as a prefix. But he can give a better explanation

Reply:
@naraandrade I've followed the same pattern used for API docs,

in this case we are getting object fields of a object definition, so the pattern for this case is: 

<method><parent-resource-name><children-resource-name-plural>

so:

getObjectDefinitionObjectFields

Reply:
I understand. My suggestion is to apply the same pattern in the function `getObjectFieldsByExternalReferenceCode` we have in the same file.

Reply:
And in that case, we would need to keep the `ById` and `ByExternalReferenceCode` to differentiate the two functions.

Reply:
got it, good catch @naraandrade

I'll change this,

Thanks!

Reply:
Done [here](https://github.com/liferay-objects/liferay-portal/pull/2571/commits/e64968b2441a64b3434cb060f8e20059c973cc93)